### ISSUE TITLE
DIM EnC: Adding a non-virtual member to an interface fails at runtime

### ISSUE THREAD

**Issue by tmat** [dotnet/roslyn#37128]
Need to investigate where exactly it fails - likely in Core CLR.

**Comment by tommcdon** [comment]
FYI @mikem8361 as this may be a runtime issue

**Comment by mikem8361** [comment]
Can you outline a simple repo?  I've tried adding a static and non-static function to an interface in a dotnet-watch test app I have and I can't repo any problems.

**Comment by mikem8361** [comment]
Actually my repro was messed up, but now I get a rude edit from Roslyn under dotnet watch when I add a function to an existing interface.

**Comment by mikem8361** [comment]
`Program.cs(82,13): error ENC0023: Adding an abstract 'method' or overriding an inherited 'method' will prevent the debug session from continuing.`

**Comment by tmat** [comment]
Yes, we currently report rude edit. I'll create a private Roslyn build that allows it.

**Comment by mikem8361** [comment]
@tmat can you point me at another (recent) Roslyn build with this enabled? Thanks.
